sample/maxmin.yl

```diff
@@ -0,0 +1,19 @@
+sope "eleyi a fun wa ni nomba to tobi julo";
```

Comment:
Hey bro. Check the test/helpers/ folder to see samples of how test cases were written for other helper functions

Reply:
Alright I will do that now

Reply:
@anoniscoding I created test cases already, 3 for each classes, kindly review